wiki/Ranking_Criteria/osu!mania/en.md

```diff
@@ -1,78 +1,202 @@
+- **Avoid anchors consisting of three or more notes.** While they are easy for players to read, longer anchors are strenuous. 
+
+Additional guidelines for *7 key Normal* difficulties: 
+- **Avoid chords with more than 3 notes.** This includes long note patterns that involve notes in more than three columns.
+Bracket patterns should be used sparingly.
+- **Long notes played with adjacent keys that are held simultaneously should be released at the same time.** Long notes with multiple release timings require coordination that players of this level cannot reasonably handle.
+- **Avoid anchors consisting of three or more notes.** While they are easy for players to read, longer anchors are strenuous. 
+
+### Hard
+
+#### Guidelines
+
+- **HP and OD values should not be higher than 8.**
+- **Note snappings of consecutive 1/8 and higher should not be used.** These can be too physically demanding for players of this level. An exception to this rule would be the usage of 3 grace notes that are at higher-end snaps (1/8, 1/12 and 1/16) as they are only used to accentuate gracing sounds.
+- **Long streams of 1/4 notes should include, at minimum, a 1/2 pause where natural to implement after every musical phrase.** These can be too physically demanding for players of this level.
+- **Chords within 1/4 streams should be used sparingly.** The density of these patterns is often too difficult for players of this skill level.
+- **Avoid using chords in the middle of 1/6 streams.** However, Using a chord at the beginning or end of a stream is fine.
+- **1/4 Jack usage is discouraged.** These can be too physically demanding for players of this level. Minijacks with ample rest time in between may be acceptable.
+- **Avoid using more than 9 consecutives notes in a trills.** The coordination and hand balance required for this pattern is too difficult for players at this skill level.
+- **Long notes should be held for at least 1/4 of a beat.**
+- **Objects directly after short-term slider velocity changes should appear at the beatmap’s normal scroll speed.** To do this, short-term slider velocity changes should only be used in empty spaces between notes and should equalize to a distance of 1.0x.
+- **Long-term slider velocity changes should be between 0.8x and 1.10x.**
+- **Slider Velocity gimmicks should be reactable within 1/2 of a beat or more.**
+
+If Hard is the *lowest difficulty of a beatmap set*, it has to abide by these additional **guidelines**:
+- **Chords within streams are discouraged.** The density of these patterns is often too difficult for players of this skill level, especially when used on the lowest difficulty of a beatmap set. However, using a chord at the beginning/end of a stream is fine.
+- **Short-term slider velocity changes should be used cautiously.** Changes should only happen for sections of different pacings and the slider velocity should not be drastically variable.
+
+Additional guidelines for *4 key Hard* difficulties: 
```

Comment:
upper half of this draft calls it 4-Key and this calls it 4 key, decide for one way?